newIDE/app/src/Utils/Mathematics.js

```diff
@@ -2,3 +2,6 @@
 
 export const roundTo = (value: number, precision: number): number =>
   Math.round(value * Math.pow(10, precision)) / Math.pow(10, precision);
+
+export const truncTo = (value: number, precision: number): number =>
+  Math.trunc(value * Math.pow(10, precision)) / Math.pow(10, precision);
```

Comment:
It probably won't work because `number` can't take any values.
https://www.exploringbinary.com/why-0-point-1-does-not-exist-in-floating-point/

Reply:
I think there's something we don't understand be cause this works:
<img width="297" alt="image" src="https://user-images.githubusercontent.com/32449369/203268981-72020b95-878c-462d-a422-56a44931af9e.png">
I guess the same would work for trunc

Reply:
Why not used [toFixed()](https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed) ? It seem easier for display.

Reply:
Problem with toFixed is that it returns a string, it's not made for operations on a number.
So it'll probably mess up the field.
I'll think about it